ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js

```diff
@@ -254,6 +248,10 @@ export function interactAndWaitForConfigurationManagementScan(interactionCallbac
         interactionCallback,
         routeMatcherMapForConfigurationManagementDashboard
     );
+
+    cy.get('div:contains("Compliance scanning in progress")', {
+        timeout: 30000,
+    }).should('not.exist');
```

Comment:
Again here the number of requests that we need to wait for is not deterministic so we wait on the UI status instead.